### DISCUSSION TITLE
Simplified notification messages?

### DISCUSSION THREAD

**Discussion by rg9400** [containrrr/watchtower#704] [Q&A] [answered]
I switched from Ouroboros to Watchtower, and I am overall happy with the decision. However, one issue I noticed is that the notification system seems to be a lot less user-friendly. Ouroboros would send a single message to my Slack/Discord agent per container update; however, Watchtower sends 4. It sends that it found a new image, stopping the current image, starting the new image, and removing the old image. With the new header response, I also get a message that it could not check the headers, and instead is falling bac to a regular pull (this is a separate issue).

In essence, this results in a ton of output. If you run the updates only on a daily basis and have multiple containers that need to be updated, you can end up with 30-40+ messages to go through. Further complicating things is that the messages are not grouped by container, so it becomes more difficult to figure out which containers were actually updated. I get why this might be useful in logging, but it sounds less useful for notification purposes, at least in my usecase. Now, I know I can change the log level from INFO, but I guess I would lose the notifications completely then. Is there a way to just get a simplified single notification per container, similar to Ouroboros?

**Comment by simonaronsson** [comment, marked as answer]
Hey!

A similar suggestion is already tracked in #255.
Something along the lines of what you're describing, or at least being able to post it as a summary, is on the long-term agenda.

Best,
Simme.

**Reply by rg9400** [reply]
A summary notification would be great, highlighting which containers were updated in a run in a single message
